### ISSUE TITLE
Automatically set Ops Permisssion via ENV var Username

### ISSUE THREAD

**Issue by 13rac1** [AngelAlvarado/dockerizing-streamsets-tutorials#1]
The server is looking for `ops.json` on startup:
```
[17:49:02] [Server thread/WARN]: Failed to load operators list: 
java.io.FileNotFoundException: ops.json (No such file or directory)
	at java.io.FileInputStream.open0(Native Method) ~[?:1.8.0_121]
	at java.io.FileInputStream.open(FileInputStream.java:195) ~[?:1.8.0_121]
	at java.io.FileInputStream.<init>(FileInputStream.java:138) ~[?:1.8.0_121]
	at com.google.common.io.Files.newReader(Files.java:86) ~[minecraft_server.1.11.2.jar:?]
	at mx.g(SourceFile:136) ~[minecraft_server.1.11.2.jar:?]
	at lg.A(SourceFile:107) [minecraft_server.1.11.2.jar:?]
	at lg.<init>(SourceFile:29) [minecraft_server.1.11.2.jar:?]
	at lh.j(SourceFile:186) [minecraft_server.1.11.2.jar:?]
	at net.minecraft.server.MinecraftServer.run(SourceFile:436) [minecraft_server.1.11.2.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
```

It just needs to be there. Details for you soonish.

**Comment by 13rac1** [comment]
`itzg/minecraft-server` supports the old style `ops.txt`, but not the new.
```
/ # cat start-minecraft | grep ops
if [ -n "$OPS" -a ! -e ops.txt.converted ]; then
  echo "Setting ops"
  echo $OPS | awk -v RS=, '{print}' >> ops.txt
```

**Comment by 13rac1** [comment]
Finally had a good excuse to learn `jq`! Check it out!

```bash
export OPS="notch,jeb"
export OPS_JSON=`echo -n "$OPS" | jq --raw-input --slurp 'split(",")' -`
wget -S -O - --header="Content-type: application/json" --post-data="$OPS_JSON" https://api.mojang.com/profiles/minecraft | jq ".[].level = 4" > ops.json
```

Result:
```bash
$ cat ops.json 
[
  {
    "id": "069a79f444e94726a5befca90e38aaf5",
    "name": "Notch",
    "level": 4
  },
  {
    "id": "f498513ce8c84773be26ecfc7ed5185d",
    "name": "jeb",
    "legacy": true,
    "level": 4
  }
]
```
*Normally, I'd do this with `curl`, but `itzg/minecraft-server` only has `wget` installed.*

**Comment by 13rac1** [comment]
Busybox on `itzg/minecraft-server ` uses a non-standard `wget`. Needs `curl` anyway. 😢 BUT! Now I can do it in one command:
```
  echo -n "$OPS" \
    | jq --raw-input --slurp 'split(",")' - \
    | curl -H "Content-type: application/json" \
    -d @- \
    https://api.mojang.com/profiles/minecraft \
    | jq ".[].level = 4" \
    > ops.json
```

**Comment by 13rac1** [comment]
Haha! I actually read the source for `start-minecraft.sh`. There's an easier method. Check #2

**Comment by AngelAlvarado** [comment]
Fixed using #2. Take a look at this new [repo](https://github.com/AngelAlvarado/docker-streamsets-kafka-minecraft)
